### DISCUSSION TITLE
WIP: Crabby - My gateway to ExcaliburJS

### DISCUSSION THREAD

**Discussion by SimponGoril** [excaliburjs/Excalibur#2360] [Show and tell]
Hi all. First of all, thank you @eonarheim many times for this **amazing framework**! Let me share my WIP project on which I am getting familiar with engine basics:

 
<img width="911" alt="image" src="https://user-images.githubusercontent.com/70171397/175280622-f758d5a1-cb60-463d-a229-3bcff5599ad0.png">

[Play it here](http://public-bucket-hosting.s3-website.eu-central-1.amazonaws.com)

_Source code coming soon!_

**Comment by eonarheim** [comment]
@SimponGoril Thanks for the kind words! And thanks for sharing this is awesome!
